src/encoded/types/output_file.py

```diff
@@ -14,3 +14,10 @@ class OutputFile(File):
     item_type = "output_file"
     schema = load_schema("encoded:schemas/output_file.json")
     embedded_list = []
+
+    # processed files don't want md5 as unique key
+    def unique_keys(self, properties):
+        keys = super(OutputFile, self).unique_keys(properties)
+        if keys.get('alias'):
```

Comment:
Should be `aliases` here? Also, rather than do this here, perhaps we should update tibanna not to patch an md5 alias instead of handling here if that's the source of the issue.

Reply:
The issue is generally for files we want md5 to be unique key, except in the case of processed files because we want to be able to rerun pipelines. This code is the same as in CGAP and I believe should work, the unit test seems to think so

Reply:
Do we need md5 to be a unique key for looking up the file? Maybe we should update `File.unique_keys` not to add that in and just do this via the schema. I don't know; this works, just kind of hacky and unclear if the complexity is necessary here.